index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Microverse Trial</title>
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+  <body>
+    <h1 id="title">Survey Form</h1>
+    <p id="description">It is a survey form for Microverse</p>
+    <form id="survey-form">
+      <label id="name-label">Name</label>
+      <input type="text" id="name" placeholder="name" required />
+
+      <label id="email-label">Email</label>
+      <input type="email" id="email" placeholder="email" required />
+
+      <label id="number-label">Number</label>
+      <input type="number" id="number" min="1" max="10" placeholder="number" />
+
+      <label for="dropdown-label">Dropdown</label>
+      <select name="dropdown" id="dropdown" required>
+        <option value="option_1">Option 1</option>
+        <option value="option_2">Option 2</option>
+      </select>
+      <div class="radio-buttons">
+        <input type="radio" name="radio" id="radio1" value="radio1" />
+        <label for="opt1">Male</label>
+        <input type="radio" name="radio" id="radio2" value="radio2" />
+        <label>Female</label>
+      </div>
+      <div class="checkboxes">
+        <input type="checkbox" value="Option 1" name="opt1" id="opt1" />
+        <input type="checkbox" value="Option 2" name="opt2" id="opt2" />
+      </div>
+      <textarea name="test" id="test" cols="30" rows="10"></textarea>
+      <button id="submit">Submit</button>
```

Comment:
- [Required] Great job for listing the comment with good reasons👍👍I think it is better to start with polite and appreciative words.👌👍
        Instead of saying:  *Your submit button should have a type * 😢👌
        It would be better to say:   *Great job ......., I recommend ......*.😁👍
- [Optional] I think it would be better to use a couple of emojis and/or gifs in the review to make the code review more human-friendly👽🤖

Reply:
Great job @Stephen-Adom👌. I recommend having a type attribute with the value submit in order to work with the form element during submission☺👍